openhtf/io/frontend/__main__.py

```diff
@@ -96,47 +96,41 @@ class StationStore(threading.Thread):
   """
   def __init__(self):
     super(StationStore, self).__init__()
-    self._death_event = threading.Event()
+    self._stop_event = threading.Event()
     self.hostname = socket.gethostname()
     self.stations = collections.defaultdict(Station)
```

Comment:
My thinking was that it's preferable to _allow_ multiple stations with the same ID by default rather than to ignore (a seemingly random) one, as long as we can provide further identifying info to disambiguate. That's why I chose to keep the unambiguous host/port pair used throughout the code here. Note also that in the new frontend UI mocks although station_id is the most prominent visual identifier, host/port are displayed as well just in case you end up with duplicates.